src/applications/vaos/newAppointmentFlow.js

```diff
@@ -61,24 +61,23 @@ export default {
           // Check if user registered systems support community care...
           const userSystemIds = await getSystemIdentifiers();
           const ccSites = await getSitesSupportingVAR();
-          const userHasCCEnabledSystems = ccSites.some(site =>
-            userSystemIds.some(userSystemId => userSystemId === site._id),
+          const ccEnabledSystems = userSystemIds.filter(id =>
```

Comment:
I changed the logic in here to save the list of CC enabled systems in the Redux state, instead of a boolean, so that I could use that list on the preferences page and not have to fetch both of these endpoints again.